### ISSUE TITLE
Update the docs codegen to inject correct examples

### ISSUE THREAD

**Issue by praneetloke** [pulumi/pulumi#4025]
* [x] Inject the correct examples instead of the hard-coded ones used in the prototype (blocked by https://github.com/pulumi/pulumi-terraform-bridge/issues/122)
* [ ] How do we handle rendering of Comment for each language?
    i.e. if the “comment” for a resource’s property contains symbols, how do we “fix-up” the symbol to have the correct casing for the specified language?

Blocked by https://github.com/pulumi/pulumi-terraform-bridge/issues/122

**Comment by praneetloke** [comment]
This issue is no longer relevant. I have separate issues open to tackle processing examples wrapped in short-codes in each language generator, as well as updating the references to types in the `Description` to be language-specific. See #4158 and #4159.
